src/components/ProfilePage/PointsBox.tsx

```diff
@@ -0,0 +1,28 @@
+import { useTheme } from '@mui/material';
+import { CenteredBox, ProfilePageTypography } from './ProfilePage.style';
+import { XPIcon } from '../illustrations/XPIcon';
+
+interface PointsBoxProps {
+  points?: number | null;
+}
+
+export const PointsBox = ({ points }: PointsBoxProps) => {
+  const theme = useTheme();
+
+  return (
+    <CenteredBox>
+      <ProfilePageTypography
+        fontSize={'80px'}
+        lineHeight={'96px'}
+        style={{
+          color: theme.palette.mode === 'light' ? '#31007A' : '#BEA0EB',
```

Comment:
We should use theme.palette colors

Reply:
There is actually a bunch of colors which we have not moved to the theme, yet. Same applies to changes within the blog. With Vilen, I came to the conclusion that we´ll have to nail things down more precisely - most new colors being used don´t have a proper name.
Same applies to the new typography components using Urbanist, each one is not grabbed from the theme, either.

@chybisov Or do you have a suggestion on what we can do until Figma setup is aligned?

Reply:
We can name them whatever we think is best suited in one place in palette until we have clear guidelines from Figma. Just to avoid a mess in the future. Depending on what colors these are, it can be for example some theme.palette.pink array with 100...900 values.